public/script.js

```diff
@@ -1484,9 +1471,6 @@ async function Generate(type, automatic_trigger, force_name2) {
         if (power_user.pin_examples && main_api !== 'openai') {
             for (let example of mesExamplesArray) {
                 if (!is_pygmalion) {
-                    if (!storyString.endsWith('\n')) {
-                        storyString += '\n';
-                    }
```

Comment:
`storyString` is now guaranteed to be properly terminated, so this is no longer needed. Moreover, it was a poor check, because it would add a useless blank line when `storyString` is empty.